packages/clay-link/src/index.tsx

```diff
@@ -59,12 +102,15 @@ const ClayLink = React.forwardRef<HTMLAnchorElement, IProps>(
 			button,
 			children,
 			className,
+			decoration,
 			displayType,
```

Comment:
Maybe we have to set the default value of displayType to `primary` to use the presets when nothing is defined?

Reply:
@matuzalemsteles I think it's better not to have a default. We have components like navigation bar and pagination that rely on `ClayLink`. If we default to primary, we will have to go back and undo them in the components. Also from a user's perspective I would expect `ClayLink` to output a standard link if I don't specify a `displayType`.

Reply:
Hmm yes, this can cause unwanted issues in other components where this is not expected. My concern is that this change is an update to fix an accessibility issue, ideally anyone using the component would not need to update or set a property to "get the fix".

But it really seems like a safer way to go that way without setting the `displayType` by default.